src/applications/personalization/dashboard-2/components/IdentityNotVerified.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
```

Comment:
This component is nearly a copy of the two IdentifyNotVerified components that exist. I'll make a ticket to create a reusable version that can be used wherever we need it.

Reply:
Ha, jinx!